libnmstate/nm/ipv4.py

```diff
@@ -67,6 +76,9 @@ def get_info(active_connection):
     if ip_profile:
         info['dhcp'] = ip_profile.get_method() == (
             nmclient.NM.SETTING_IP4_CONFIG_METHOD_AUTO)
+        info['ignore-auto-routes'] = ip_profile.props.ignore_auto_routes
```

Comment:
If it was never set, won't it return `None`?

Should we return these if `dhcp` is disabled?

Reply:
> If it was never set, won't it return `None`?
The [libnm GI document][1] said it's a bool:

[1]: https://lazka.github.io/pgi-docs/#NM-1.0/classes/SettingIPConfig.html#NM.SettingIPConfig.get_ignore_auto_routes
> 
> Should we return these if `dhcp` is disabled?

That goes to be a debate. Including DHCP/Autoconf options when DHCP/Autoconf is disabled requires extra user changes in yaml/json config when user simply want to enable DHCP(which happens to be most common use case).